share/validate-product-config/product-config-schema.rnc

```diff
@@ -399,9 +490,13 @@ ds.deliverable =
   }
 
 ds.deliverable_subdeliverable =
+  ## An individual DAPS-compatible document that is a DocBook set/book that includes articles/book (default language)
   element deliverable {
+    ## Enable DocBook remarks in this document (`daps ... --remarks`)
     attribute remarks { xsd:boolean }?,
+    ## Enable DocBook draft watermark in this document (`daps ... --draft`)
     attribute draft { xsd:boolean }?,
+    ## Enable DocBook meta information in this document (`daps ... --meta`)
     attribute meta { xsd:boolean }?,
```

Comment:
For `--remarks` and `--meta`, it's currently always `false`.

For `--draft`, this may be true or false depending on the lifecycle of the docset: `unpublished` = true, `beta` = currently false/ soon true again, `supported` & `unsupported` = false